README.md

```diff
@@ -1,14 +1,12 @@
-# The SRS Book by Google
-
-Google wrote a book aiming to share knowledge about our experience in building
-secure systems at scale. This highlights technologies and practices that protect
-user data and reliability. It also offers insights into collaboration between
-teams on these topics.
+# Building Secure and Reliable Systems - a book by Google
 
 This repository contains the content of the book "Building Secure and Reliable
 Systems" by Heather Adkins, Betsy Beyer, Paul Blankinship, Piotr Lewandowski,
 Ana Oprea, and Adam Stubblefield.
 
-The book was launched on April 8, 2020 at https://landing.google.com/sre/books.
+Google wrote this book aiming to share knowledge about our experience in building
+secure systems at scale. The book highlights technologies and practices that protect
+user data and reliability; it also offers insights into collaboration between
+teams on these topics. The book was launched on 2020-04-08 at https://landing.google.com/sre/books.
```

Comment:
Although the URL still resolves, is it appropriate to update the URL to https://sre.google/books ?

Reply:
Good idea, done.